src/supporting-data/locations/components/results/LocationsCard.tsx

```diff
@@ -0,0 +1,54 @@
+import { Card } from 'franklin-sites';
+import { FC, useCallback, MouseEvent } from 'react';
+import { useHistory } from 'react-router-dom';
+
+import { getEntryPath } from '../../../../app/config/urls';
+
+import { LocationsAPIModel } from '../../adapters/locationsConverter';
+import { Namespace } from '../../../../shared/types/namespaces';
+
+const BLOCK_CLICK_ON_CARD = new Set(['A', 'INPUT', 'BUTTON']);
+
+const CitationCard: FC<{
+  data: LocationsAPIModel;
+  selected?: boolean;
+  handleEntrySelection?: (rowId: string) => void;
+}> = ({ data, selected, handleEntrySelection }) => {
+  const history = useHistory();
+
+  const handleCardClick = useCallback(
+    (event: MouseEvent) => {
+      if (BLOCK_CLICK_ON_CARD.has((event.target as HTMLElement).tagName)) {
+        return;
+      }
+      history.push(getEntryPath(Namespace.locations, data.id));
+    },
+    [history, data.id]
+  );
+
+  return (
+    <Card onClick={handleCardClick}>
+      <section className="result-card">
+        {handleEntrySelection && (
+          <div className="result-card__left">
+            <input
+              type="checkbox"
+              checked={selected}
+              onChange={() => handleEntrySelection(data.id)}
+              data-testid="up-card-checkbox"
+            />
+          </div>
+        )}
+        <div className="result-card__right">
+          <h5>{data.name}</h5>
+          <div className="result-card__info-container">{data.definition}</div>
+          <div className="result-card__info-container">
+            <strong>Category:</strong> {data.category}
```

Comment:
I know there's only 1, but think we should be using `RenderColumnInCard` whenever we can.

Reply:
See https://github.com/ebi-uniprot/uniprot-website/pull/319

Reply:
See this https://github.com/ebi-uniprot/uniprot-website/blob/417d7b035c06a1fa437515c49f72b2d81f3611b8/src/supporting-data/locations/components/results/LocationsCard.tsx#L54 more specifically

Reply:
Aaaaah

Reply:
This is specifically this change that made me enter a rabbit hole of changes that led to that other PR